app/models/event.rb

```diff
@@ -1,3 +1,6 @@
 class Event < ApplicationRecord
   has_one_attached :logo
+  has_one :event_info, dependent: :destroy, foreign_key: :year,
+    primary_key: :year, inverse_of: :event
+  accepts_nested_attributes_for :event_info
```

Comment:
is it possible to accept more that one event_info using this `accepts_nested_attributes_for`?

Reply:
I have done some testing and it looks it is not possible. It must be because the relationship has been configured as a one-to-one relationship